saleor/discount/models.py

```diff
@@ -392,4 +396,4 @@ class OrderDiscount(models.Model):
     class Meta:
         # Orders searching index
         indexes = [GinIndex(fields=["name", "translated_name"])]
-        ordering = ("pk",)
+        ordering = ("created_at",)
```

Comment:
Ordering should have a unique field on the last position. Without it, pagination could not work correctly.